requirements-base.txt

```diff
@@ -18,10 +18,12 @@ email-reply-parser>=0.2.0,<0.3.0
 enum34>=1.1.6,<1.2.0
 functools32>=3.2.3,<3.3
 futures>=3.2.0,<4.0.0
-# optional
-google-cloud-bigtable>=0.32.1,<0.33.0
 # optional but needed for google stuff below
+google-cloud-bigquery==1.10.0
```

Comment:
We shouldn't include bigquery. This has no bearing on open source users.

Reply:
It's also arguable that we shouldn't need it in getsentry either. But I'm not sure if we can just yet.